marklogic-data-hub-central/ui/src/components/header/header.test.tsx

```diff
@@ -15,36 +15,60 @@ describe('Header component', () => {
 
     const { getByText, getByLabelText, queryByText } = render(
       <Router>
-        <UserContext.Provider value={userNotAuthenticated}>
-          <Header environment={data.environment}/>
-        </UserContext.Provider>
+        <Header environment={data.environment}/>
       </Router>
     )
-
+    
     expect(getByLabelText('header-logo')).toBeInTheDocument();
     expect(getByText(Application.title)).toBeInTheDocument();
     expect(getByLabelText('icon: question-circle')).toBeInTheDocument();
     expect(queryByText('icon: user')).not.toBeInTheDocument();
-
+    //unauthenticated users get sent to default Marklogic docs 
+    expect(document.querySelector('#help-link')).toHaveAttribute('href', 'https://docs.marklogic.com/datahub/')
   });
 
   test('should render correctly when a user is logged in', async () => {
 
     const { getByText, getByLabelText } = render(
       <Router>
         <UserContext.Provider value={userAuthenticated}>
-          <Header environment={data.environment}/>
+          <Header environment = {{...data.environment, dataHubVersion: '5.3-SNAPSHOT'}}/>
         </UserContext.Provider>
       </Router>
     )
-
     expect(getByLabelText('header-logo')).toBeInTheDocument();
     expect(getByText(Application.title)).toBeInTheDocument();
     expect(getByText(data.environment.serviceName)).toBeInTheDocument();
     // expect(getByLabelText('icon: search')).toBeInTheDocument();
     expect(getByLabelText('icon: question-circle')).toBeInTheDocument();
+    //test version specific link is correct when environment hub version data is set to '5.3-SNAPSHOT'
+    expect(document.querySelector('#help-link')).toHaveAttribute('href', 'https://docs.marklogic.com/datahub/5.3');
     // expect(getByLabelText('icon: setting')).toBeInTheDocument();
     expect(getByLabelText('icon: user')).toBeInTheDocument();
   });
 
-});
+  test('Verify proper version link given specific release dataHub release version', async () => {
+    const {} = render(
+      <Router>
+        <UserContext.Provider value={userAuthenticated}>
+          <Header environment = {{...data.environment, dataHubVersion: '5.2.1'}}/>
+        </UserContext.Provider>
+      </Router>
+    )
+    expect(document.querySelector('#help-link')).toHaveAttribute('href', 'https://docs.marklogic.com/datahub/5.2');
+
+  });
+
+  test('Verify proper version link given scenario with multi-digit dataHub versions', async () => {
+    const {} = render(
+      <Router>
+        <UserContext.Provider value={userAuthenticated}>
+          <Header environment = {{...data.environment, dataHubVersion: '5.64.123456'}}/>
+        </UserContext.Provider>
```

Comment:
This is ok for now. Next time you make a PR, pls try and use `rerender` to test with different variable options in the same test.

Reply:
Thanks @bsrikan . Was looking for a way to do that, did not know about rerender. Will refactor and sneak that into my next PR.